apps/api/src/app/vercel-integration/vercel-integration.module.ts

```diff
@@ -0,0 +1,13 @@
+import { HttpModule } from '@nestjs/axios';
+import { Module } from '@nestjs/common';
+import { USE_CASES } from './usecases';
+import { VercelIntegrationController } from './vercel-integration.controller';
+import { SharedModule } from '../shared/shared.module';
+import { EnvironmentsModule } from '../environments/environments.module';
+
+@Module({
+  imports: [SharedModule, HttpModule, EnvironmentsModule],
+  providers: [...USE_CASES],
+  controllers: [VercelIntegrationController],
+})
+export class VercelIntegrationModule {}
```

Comment:
My only question here would be, should we create an entire module for the vercel integration? Or maybe just add it to one of the existing modules such as auth or integrations. 

The functionality here will be quite simple and I don't see a scenario if it will actually grow in usage in the future. And by looking at the high level module structure people might get confused on what it actually is, even tho 100% of the value here is relevant only to our cloud offering. @p-fernandez @davidsoderberg @djabarovgeorge what do you think?